### DISCUSSION TITLE
DVR scan says error input file does not exist.

### DISCUSSION THREAD

**Discussion by eremika0000** [Breakthrough/DVR-Scan#184] [Q&A]
I am very new to this but I keep getting this error when inputting the file name into cmd. I open cmd and type C:\Users\eremi>dvr-scan.exe -i PXL_20241214_203950764.TS

And i get the error

[DVR-Scan] DVR-Scan 1.6.1
[DVR-Scan] Error: Input file does not exist:
  PXL_20241214_203950764.TS

How can I fix this? It is an mp4 file i'm trying to scan. Does it matter the video is in my downloads folder. Is there any other commands I was supposed to type first? Reading the dvr scan guide I thought as soon as I open cmd I type dvr-scan i- (file name). I download it from the windows section of the downloads page. Was there something else I was supposed to do other than running the installer and then running cmd right away

**Comment by Breakthrough** [comment]
See #178 for some handy shortcut files to use DVR-Scan without a command prompt.  (If you use these, please provide some feedback so I can know if they should be included in the official release! 👍 )

When using the command prompt, you specify a full path to the input file, otherwise DVR-Scan looks at the current folder your command prompt is in.  Try to `cd` into your downloads folder and run DVR-Scan from there, or provide an absolute path as the input.

Either of the following should work:

    cd C:\Users\eremi\Downloads
    dvr-scan -i PXL_20241214_203950764.TS

Or from any folder:

    dvr-scan -i C:\Users\eremi\Downloads\PXL_20241214_203950764.TS

Also, see #178 which includes a link to a .ZIP file containing some helper shortcuts so you can drag-and-drop files to run DVR-Scan.  I'm working on a GUI for DVR-Scan so files can be selected with a file browser, but this is still in development.  Stay tuned for a future version that includes it :)

Let me know if you run into any other issues. Note that the output will be by default in the current working directory (whatever you `cd` to).  You can also specify an output directory with `-d`/`--output-dir`.

**Reply by Breakthrough** [reply]
If you end up using the .bat files in the ZIP I linked, please let me know if they work well for you.  They should allow you to use DVR-Scan without a command prompt.  I can include these with the installer if they work well, but I haven't gotten any feedback on them yet unfortunately.

The next release of DVR-Scan (v1.6.2) will include a much better region editor, and after that, it will be transitioned into a full GUI for v1.7  When that's done, the installer will include a desktop shortcut to launch the app.

**Reply by eremika0000** [reply]
Sorry I am still confused.  What do you mean try to cd into your downloads folder and run dvr scan from there? Wasn't sure what cd was. And I did try to download the bat files but chrome and edge kept saying it had a virus so I wasn't able to complete the download. I'm new to using this type of program sorry. I try putting cd C:\Users\eremi\Downloads into cmd and then presssed enter and then dvr-scan -i PXL_20241214_203950764.TS but I got the same error saying error input file doesn't exist

Update: Nevermind it seems once I added .mp4 at the end it was able to recognize it. thanks.

**Reply by Breakthrough** [reply]
> Sorry I am still confused. What do you mean try to cd into your downloads folder and run dvr scan from there? Wasn't sure what cd was. And I did try to download the bat files but chrome and edge kept saying it had a virus so I wasn't able to complete the download. I'm new to using this type of program sorry. I try putting cd C:\Users\eremi\Downloads into cmd and then presssed enter and then dvr-scan -i PXL_20241214_203950764.TS but I got the same error saying error input file doesn't exist
> 
> Update: Nevermind it seems once I added .mp4 at the end it was able to recognize it. thanks.

Thanks for trying, it might be flagging that as a virus since it contains .BAT files.  Sorry about that.

And yes unfortunately DVR-Scan needs the full path to the file including the extension, sometimes it is not shown in Windows explorer.  One handy thing you can do with command prompt is use the tab key to auto-complete filenames, so if you type PXL_ and hit `tab`, it should auto-complete as much of the remainder of the filename as it can.

**Reply by eremika0000** [reply]
I see thank you, I got it to scan the file. I wanted to know what command would I use to make it more sensitive to motion? I think I saw that there was an option to change the sensitivity and would i have to execute this command each time I reopen cmd? And what command would I use to use to scan multiple files from the downloads folder? Thanks for your help

And if it did detect any motion from the videos they would save as a new video into the downloads folder ? Or program files?

**Reply by Breakthrough** [reply]
I've updated the instructions in #178 so you don't need to download any files, you can create the helper scripts yourself easily with notepad.  This will make it easier to scan multiple files since you can just drag-and-drop them onto the helper script.

#178 also includes instructions in how to create a config file so you can tweak detection settings.  You probably want to start with the "threshold" option, lower is more sensitive to motion, higher is less sensitive.

The default is `threshold = 0.15` which means 15% of the frame must contain motion.

**Reply by Breakthrough** [reply]
>And what command would I use to use to scan multiple files from the downloads folder?

You can specify multiple file paths after `-i` or use the helper script in #178 to drag and drop.

> And if it did detect any motion from the videos they would save as a new video into the downloads folder ? Or program files?

They will be output from the current working directory by default, so the path that shows up in your command prompt window when you run the command.  You can customize this with a config file by setting `output-dir = C:\output_folder` in a config file, or by adding `-d C:\output_folder` to your command.

Edit: thanks for the feedback as well, it's quite valuable to hear from users like this.  I'll be sure to keep these questions and issues you've ran into in mind as I work on improving the user experience.  Happy to help out as well if you run into any issues with scanning or the config file.  Right now you need to create one manually as per #178, but in the next release I'll make sure the installer creates a default config file for you.

**Reply by eremika0000** [reply]
Okay I created the config file, just wanted to make sure I can just name it config and copy and paste all the text from the config file template? Also you said I can drag the video files I want to scan onto the .bat created from The 178 instructions ? The bat should only have
```
ECHO OFF
dvr-scan.exe -i %*
PAUSE
```
Correct ? And I just drag the viedeos on top of the bat file ? Sorry wanted to make sure I'm doing it right

**Reply by Breakthrough** [reply]
> Okay I created the config file, just wanted to make sure I can just name it config and copy and paste all the text from the config file template?

For DVR-Scan to find a config file automatically, it must be named `dvr-scan.cfg` (no .txt extension) and located in the `C:/Users/%USERNAME%/AppData/Local/DVR-Scan/` folder, where %USERNAME% is your username (so eremi in your case).

You should see a log message when you run the program that tells you when it is using the file.  You can add the "-so" flag (scan-only) to make sure no output is generated if you want to test things out, and hit Ctrl + C at any time to stop processing.

**Reply by Breakthrough** [reply]
>You said I can drag the video files I want to scan onto the .bat created from The 178 instructions ? The bat should only have
> 
> ```
> ECHO OFF
> dvr-scan.exe -i %*
> PAUSE
> ```
> 
> Correct ? And I just drag the viedeos on top of the bat file ? Sorry wanted to make sure I'm doing it right

Correct 😄

**Reply by eremika0000** [reply]
Please I wanted to know the best settings if I wanted DVR scan to be able to catch very slight motion that lasts only less than a second in a black and grey camera because it is a night vision camera. I tried  the default settings and DVR scan didn't detect any motion so I tried putting the threshhold to 0.02 and min event length to 0.1, but after saving and running the video through DVR scan it did not find any motion again. aI did watch through the video to make sure there was some slight motion and there was. Perhaps I didn't save the config properly? I simply changed the number and saved the txt doc but didn't change anything else other than the value. 

I am using this command to scan with region editor not sure if it matters. The command  ECHO OFF
dvr-scan.exe -i %*
PAUSE

**Reply by Breakthrough** [reply]
> Please I wanted to know the best settings if I wanted DVR scan to be able to catch very slight motion that lasts only less than a second in a black and grey camera because it is a night vision camera. I tried the default settings and DVR scan didn't detect any motion so I tried putting the threshhold to 0.02 and min event length to 0.1, but after saving and running the video through DVR scan it did not find any motion again. aI did watch through the video to make sure there was some slight motion and there was. Perhaps I didn't save the config properly? I simply changed the number and saved the txt doc but didn't change anything else other than the value.

Lines starting with a `#` symbol are ignored as comments, make sure the lines where you set the values do not start with a `#` symbol, e.g your config file should have:

    #lines starting with # are ignored
    threshold = 0.02
    min-event-length = 0.1s

Feel free to post your full config file if you run into any issues.

----------------------------

You can also enable debug logging by changing your bat file to:

```
ECHO OFF
dvr-scan.exe -v debug -i %*
PAUSE
```

You should then see the values that get loaded from the config file near the top of the output. With the config file values above, you should see something like this at the beginning of running DVR-Scan (you should see your config file values after "Loaded configuration"):

```
INFO: controller.parse_settings(): DVR-Scan 1.61
INFO: controller.parse_settings(): Loading user config file:
  C:\Users\Breakthrough\AppData\Local\DVR-Scan\dvr-scan.cfg
DEBUG: controller.parse_settings(): Loaded configuration:
{'min-event-length': 0.1s, 'threshold': 0.02}
```

I also filed a bug #186 to improve logging here so that these values are printed normally without debug logging, since they are pretty important to see when you are changing them.

**Comment by eremika0000** [comment]
Okay I did change the config file to this and tried rescanning the video but it still didn't catch the slight motion in the video did I do something wrong to the config? I tried changing the bat to the text you provided but I got this message back. I see it says config file not found for some reason. My config is called dvr-scan.cfg and it is located in C:\Users\eremi\AppData\Local\DVR-Scan so I'm not sure why it was not found. I also post my config file. I was thinking maybe it is not detecting the slight motion because the config file isn't working since it cant find the config?
```
 C:\Users\eremi\Downloads>ECHO OFF
INFO: controller.parse_settings(): DVR-Scan 1.6.1
DEBUG: controller.parse_settings(): User config file not found.
DEBUG: controller.parse_settings(): Program arguments:
Namespace(scan_only=False, debug=False, verbosity='debug', input=['C:\\Users\\eremi\\Downloads\\5to530.mp4'])
INFO: controller.run_dvr_scan(): Initializing scan context...
INFO: video_joiner._load_input_videos(): Opened video 5to530.mp4 (2304 x 1296 at 15.000 FPS).
DEBUG: scanner._handle_regions(): No regions selected.
INFO: scanner.scan(): Using subtractor MOG2 with kernel_size = 7 (auto) and learning_rate = auto
INFO: scanner.scan(): Scanning input video for motion events...
Detected: 0 | Progress:   5%|██▍ 
```

```
#
# This file contains all possible DVR-Scan config options. Lines beginning
# with a # are treated as comments and ignored. Remove them when setting a
# config value.
#
# A config file path can be specified via the -c/--config option.
# DVR-Scan also looks for a `dvr-scan.cfg` file in the following locations:
#
#   Windows: C:/Users/%USERNAME%/AppData/Local/DVR-Scan/dvr-scan.cfg
#
#   Linux:   ~/.config/DVR-Scan/dvr-scan.cfg
#            $XDG_CONFIG_HOME/dvr-scan.cfg
#
#   Mac:     ~/Library/Preferences/DVR-Scan/dvr-scan.cfg
#
# Run `dvr-scan --help` to see the exact path on your system which will be
# used (it will be listed under the help text for the -c/--config option).
#

# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
#  GENERAL
# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *

# Show region editor window (-r/--region-editor) before scanning.
#region-editor = no

# Suppress all console output.
#quiet-mode = no

# Verbosity of console output (debug, info, warning, error).
# If set to debug, overrides quiet-mode unless set via command line.
#verbosity = info


# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
#  INPUT / OUTPUT
# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *

# Directory to output all created files. If unset, files will be created
# in the current working directory.
#output-dir = /usr/tmp/somedir

# Method of generating output videos. Possible values are: SCAN_ONLY, OPENCV,
# FFMPEG, COPY. Not all features are supported in FFMPEG/COPY mode.
#output-mode = OPENCV

# Arguments to add before the input when calling ffmpeg in output-mode FFMPEG
# or COPY. Note that `-y` and `-nostdin` are always added.
#ffmpeg-input-args = -v error

# Encoder parameters used when generating output files in FFMPEG mode.
#ffmpeg-output-args = -map 0 -c:v libx264 -preset fast -crf 21 -c:a aac -sn

# Four-letter identifier of the encoder/video codec to use in OPENCV mode.
# Possible values are: XVID, MP4V, MP42, H264
#opencv-codec = XVID


# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
#  MOTION EVENTS
# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *

Amount of time which must have motion in each frame to trigger an event.
min-event-length = 0.1

Amount of time to include before an event.
time-before-event = 1.5s

Amount of time to include after an event.
time-post-event = 2.0s


# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
#  DETECTION PARAMETERS
# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *

# Type of background subtraction to use, one of: MOG2, CNT, MOG2_CUDA
#bg-subtractor = MOG2

Threshold representing amount of motion in a frame (or the ROI, if set) for
a motion event to be triggered. Lower values require less movement, and are
more sensitive to motion. If too high, some movement may not be detected,
while too low of a threshold can result in false detection events.
threshold = 0.01

# Scores of this amount or higher are ignored. 255.0 is the maximum score, so
# values greater than 255.0 will disable the filter.
#max-threshold = 255.0

# Size (in pixels) of the noise reduction kernel. Can be odd integer starting
# from 3, 0 to disable, or -1 to auto-set using video resolution.
#kernel-size = -1

# Region file to limit detection areas. Can be created with the -r / --region-editor
# See < http://www.dvr-scan.com/guide/#region-editor > for an overview of the editor.
#load-region = roi.txt

# Integer factor to shrink video before processing. Values <= 1 have no effect.
#downscale-factor = 0

# Number of frames to skip between processing when looking for motion events.
#frame-skip = 0


# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
#  OVERLAYS
# * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *

#
# Text Overlays
#

# Enable/disable timecode overlay in top-left of video frame.
#time-code = yes

# Enable/disable frame metrics overlay in top right of video frame.
#frame-metrics = yes

# Size of border around text, in pixels. Must be <= text-margin.
#text-border = 4

# Size of margin around border, in pixels.
#text-margin = 4

# Font scale factor.
#text-font-scale = 2.0

# Thickness of font (integer values only).
#text-font-thickness = 2

# Text color in the form (R,G,B) or 0xFFFFFF
#text-font-color = 255, 255, 255

# Text background color in the form (R,G,B) or 0xFFFFFF
#text-bg-color = 0, 0, 0


#
# Bounding Box Overlay
#

# Enable/disable overlay.
#bounding-box = yes

# Box edge color in the form (R,G,B) or 0xFFFFFF
#bounding-box-color = 255, 0, 0

# Thickness of bounding box, relative to largest edge of input video.
#bounding-box-thickness = 0.0032

# Amount of temporal smoothing to apply.
#bounding-box-smooth-time = 0.1s

# Minimum side length of bounding box, relative to largest edge of input video.
#bounding-box-min-size = 0.032
```

**Reply by Breakthrough** [reply]
Your config file looks correct.  Indeed it seems DVR-Scan cannot find your config file. Can you double check the file extension?  The file icon should look like a gear in Windows Explorer, for example:

![image](https://github.com/user-attachments/assets/f15bdf6a-8bb0-4b7e-a396-d71032383fe7)

If it looks like a piece of paper instead, it might end with `.txt` which Windows Explorer hides by default.  You can make it show the extensions by going View -> Options and unchecking "Hide extensions for known file types" under Folder Options -> View:

![image](https://github.com/user-attachments/assets/397b3425-e518-4fd0-8c17-4e3336594e6f)

![image](https://github.com/user-attachments/assets/8e12e781-4543-42b9-aa30-6a8b39983350)

If that was not the problem, could you check the output of running `dvr-scan.exe --help` in a command prompt?  Under the help entry for `-c`/`--config`, it should show you the exact path it looks for your config file at.  Please verify that is correct and your config file is in that exact location.  If you are unsure feel free to copy/paste the output of that command here.

If for some reason that does not work, please file a bug report so I can fix the issue for the upcoming release.  If the config file continues to give you issues, you can also set the params via your .bat file, e.g.:

```
ECHO OFF
dvr-scan.exe -v debug -i %* --threshold 0.01 --min-event-length 0.1s
PAUSE
```

-----------------

Aside, thanks again for sharing your experience, this has been a huge help for improving DVR-Scan.  I apologize for how difficult this has been, and will do my best to resolve these usability issues going forwards now that you have made me aware of them.  The next release of DVR-Scan will create an empty user config file for you in the correct location so folks can just open and edit it without running into any issues like this.  I hope to get that done within the next day or two.

**Reply by eremika0000** [reply]
Thanks for the help so far. So it seems the config file was a .txt, so I renamed the file and got rid of the .txt so now it is a cfg file. When trying to scan a video through the bat  I get this messages in command prompt:

```
C:\Users\eremi\Downloads>ECHO OFF
INFO: controller.parse_settings(): DVR-Scan 1.6.1
INFO: controller.parse_settings(): Loading user config file:
  C:\Users\eremi\AppData\Local\DVR-Scan\dvr-scan.cfg
ERROR: controller.parse_settings(): Error: Source contains parsing errors: 'C:\\Users\\eremi\\AppData\\Local\\DVR-Scan\\dvr-scan.cfg'
  [line 64]: 'Amount of time which must have motion in each frame to trigger an event.\n'
  [line 67]: 'Amount of time to include before an event.\n'
  [line 70]: 'Amount of time to include after an event.\n'
  [line 81]: 'Threshold representing amount of motion in a frame (or the ROI, if set) for\n'
  [line 82]: 'a motion event to be triggered. Lower values require less movement, and are\n'
  [line 83]: 'more sensitive to motion. If too high, some movement may not be detected,\n'
  [line 84]: 'while too low of a threshold can result in false detection events.\n'
CRITICAL: controller.parse_settings(): Failed to load config file.
DEBUG: controller.parse_settings(): Error loading config file:
Traceback (most recent call last):
  File "dvr_scan\cli\config.py", line 422, in load
  File "configparser.py", line 723, in read_string
  File "configparser.py", line 718, in read_file
  File "configparser.py", line 1116, in _read
configparser.ParsingError: Source contains parsing errors: 'C:\\Users\\eremi\\AppData\\Local\\DVR-Scan\\dvr-scan.cfg'
        [line 64]: 'Amount of time which must have motion in each frame to trigger an event.\n'
        [line 67]: 'Amount of time to include before an event.\n'
        [line 70]: 'Amount of time to include after an event.\n'
        [line 81]: 'Threshold representing amount of motion in a frame (or the ROI, if set) for\n'
        [line 82]: 'a motion event to be triggered. Lower values require less movement, and are\n'
        [line 83]: 'more sensitive to motion. If too high, some movement may not be detected,\n'
        [line 84]: 'while too low of a threshold can result in false detection events.\n'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "dvr_scan\cli\controller.py", line 139, in parse_settings
  File "dvr_scan\cli\controller.py", line 127, in parse_settings
  File "dvr_scan\cli\config.py", line 424, in load
dvr_scan.cli.config.ConfigLoadFailure
Press any key to continue . . .
```

I also try putting dvr-scan.exe --help into cmd and I get this message

```
C:\Users\eremi>dvr-scan.exe --help
'dvr-scan.exe' is not recognized as an internal or external command,
operable program or batch file.
```

I do think my cfg file is in the right location. The address for it is C:\Users\eremi\AppData\Local\DVR-Scan
So I tried deleting the cfg file and copying this onto my bat and it works great now thank you

```
ECHO OFF
dvr-scan.exe -v debug -i %* --threshold 0.01 --min-event-length 0.1s
PAUSE
```

**Reply by Breakthrough** [reply]
Glad you finally got things working!  And sorry I realize now there *was* an issue with your config file: only config values can be present in the file, all other lines *must* start with #.  For example, you must change:

```
Threshold representing amount of motion in a frame (or the ROI, if set) for
a motion event to be triggered. Lower values require less movement, and are
more sensitive to motion. If too high, some movement may not be detected,
while too low of a threshold can result in false detection events.
threshold = 0.01
```

To either put # infront of each comment line, or delete them:

```
# Threshold representing amount of motion in a frame (or the ROI, if set) for
# a motion event to be triggered. Lower values require less movement, and are
# more sensitive to motion. If too high, some movement may not be detected,
# while too low of a threshold can result in false detection events.

threshold = 0.01
```

**Comment by eremika0000** [comment]
Today I got this strange error twice when using dvr scan

C:\Users\eremi\Downloads>ECHO OFF
INFO: controller.parse_settings(): DVR-Scan 1.6.1
DEBUG: controller.parse_settings(): User config file not found.
DEBUG: controller.parse_settings(): Program arguments:
Namespace(scan_only=False, debug=False, verbosity='debug', input=['C:\\Users\\eremi\\Downloads\\241230_05.00.04_05.30.04_S.avx.mp4', 'C:\\Users\\eremi\\Downloads\\241230_07.00.08_07.30.08_S.avx.mp4', 'C:\\Users\\eremi\\Downloads\\241230_06.30.07_07.00.07_S.avx.mp4', 'C:\\Users\\eremi\\Downloads\\241230_05.30.05_06.00.05_S.avx.mp4'], threshold=0.01, min_event_length=0.1)
INFO: controller.run_dvr_scan(): Initializing scan context...
INFO: video_joiner._load_input_videos(): Opened video 241230_05.00.04_05.30.04_S.avx.mp4 (2304 x 1296 at 15.000 FPS).
INFO: video_joiner._load_input_videos(): Appending video 241230_07.00.08_07.30.08_S.avx.mp4 (2304 x 1296 at 15.000 FPS).
INFO: video_joiner._load_input_videos(): Appending video 241230_06.30.07_07.00.07_S.avx.mp4 (2304 x 1296 at 15.000 FPS).
INFO: video_joiner._load_input_videos(): Appending video 241230_05.30.05_06.00.05_S.avx.mp4 (2304 x 1296 at 15.000 FPS).
DEBUG: scanner._handle_regions(): No regions selected.
INFO: scanner.scan(): Using subtractor MOG2 with kernel_size = 7 (auto) and learning_rate = auto
INFO: scanner.scan(): Scanning 4 input videos for motion events...
DEBUG: video_joiner.read(): End of current video, loading next: C:\Users\eremi\Downloads\241230_07.00.08_07.30.08_S.avx.mp4
DEBUG: video_joiner.read(): End of current video, loading next: C:\Users\eremi\Downloads\241230_06.30.07_07.00.07_S.avx.mp4
Detected: 0 | Progress:  75%|██████████████████████████████████▍           | 83340/111154 [30:45<09:09, 50.65 frames/s][hevc @ 000001c6def349c0] PTL information too short
DEBUG: video_joiner.read(): End of current video, loading next: C:\Users\eremi\Downloads\241230_05.30.05_06.00.05_S.avx.mp4
DEBUG: video_joiner.read(): No more input to process.
Detected: 0 | Progress: 100%|█████████████████████████████████████████████| 111154/111154 [46:20<00:00, 39.98 frames/s]
INFO: controller.run_dvr_scan(): Processed 111154 frames read in 2780.2 secs (avg 40.0 FPS).
INFO: controller.run_dvr_scan(): No motion events detected in input.
Press any key to continue . . .


Why does it say ptl information is too short?  Does it mean it scanend my videos improperly? I have been using the dvr scan bat file almost every day but not until today did I start getting this message

**Reply by Breakthrough** [reply]
> Why does it say ptl information is too short? Does it mean it scanend my videos improperly? I have been using the dvr scan bat file almost every day but not until today did I start getting this message

This means an error happened when DVR-Scan was parsing that video.  The message comes from `ffmpeg` which DVR-Scan uses to read each frame from the video.  It's possible the video was corrupted or something happened to the recording device.  How long does the final video appear to be when you open and play it back manually?  It looks like the last video stopped being scanned around the 15 minute mark, but the video says it should be 30 minutes.

There's [a FAQ entry](https://dvr-scan.readthedocs.io/en/latest/faq/#how-can-i-fix-a-video-that-plays-but-cannot-be-scanned) that covers this, or you can try re-encoding the video with another tool before processing it.  There's a few reasons why this can occur, but typically if you can play the video with another player like VLC, it can usually be fixed in this way.

**Reply by eremika0000** [reply]
I see thank you, I wanted to know is there any way I can view an error log file ? Do I have to input a command into the bat file? I got an error file about ptl information being short and accidentally closed the file. Wanted to know if there's a log i can view to check the error because I forgot the time of the error 

Also I tend to scan multiple videos at once, if it says just one video had errors should the other videos scanned be okay too or should I restart the scans without the video which had error ?

**Reply by Breakthrough** [reply]
is there any way I can view an error log file ? Do I have to input a command into the bat file?

If you add `--logfile log.txt` before the `-v debug` part of the DVR-Scan command, it will save the logs to `log.txt`.

> Also I tend to scan multiple videos at once, if it says just one video had errors should the other videos scanned be okay too or should I restart the scans without the video which had error ?

It depends on the nature of the error, but DVR-Scan should indicate in the log messages if it's completed the rest of the videos or halted.  Feel free to upload a logfile here and I can provide further help if it's not clear.

---------------

Unrelated: there's a new GUI version of DVR-Scan currently in beta, [see this link](https://github.com/Breakthrough/DVR-Scan/issues/198) if you are interested in testing it out.  Thanks.

**Reply by eremika0000** [reply]
Lately when trying to delete  avi files from dvr scan that it created,  I get the rror that the file cannot be deleted because it is open in shell infrastructure host. It happens often and I have to repetedly go end that task in task manager. Do you know why this happens or how I can fix? thanks.  Thanks for the help
